ReactiveCocoaTests/Swift/SignalLifetimeSpec.swift

```diff
@@ -26,37 +26,41 @@ class SignalLifetimeSpec: QuickSpec {
 				expect(signal).to(beNil())
 			}
 
-			it("should deallocate even if it has an observer") {
+			it("should deallocate if it does not have any observers") {
```

Comment:
*should not deallocate if it has at least one observer* should be the test case you want.